.gitignore

```diff
@@ -2,6 +2,9 @@
 ios/Flutter/flutter_export_environment.sh
 .idea/**
 
+# MacOS generated file
+.DS_Store
```

Comment:
Removed such file from repo and added to `gitignore` as it is not something that has to be in the repo.